### PR TITLE
Add support for running with relocatable

### PR DESCRIPTION
**Pull request by fruch**
running with scylla-ccm and relocatable packages, needs
some change to the way ccm command line is being called

`scylla_version` and `install_dir` are mutually exclusive,
or you can work from the source directory of scylla,
or you can work with relocatable package.

you can't do both at the same time.

Moved some parameters into the create command, since scylla-ccm support for relocatable
isn't working when a separated populate command line is issued.

### CODE REVIEW

File: tests/src/integration/ccm/bridge.cpp [hunk 1]
```diff
@@ -385,6 +399,8 @@ bool CCM::Bridge::create_cluster(std::vector<unsigned short> data_center_nodes,
     }
 
     // Create the cluster populate command and execute
+    /* Moved those parameters into the create command, since scylla-ccm doesn't support relocatable
```
Comment (jul-stas) [inline tests/src/integration/ccm/bridge.cpp:402]: Perhaps we should remove it instead of commenting out?

**Comment by jul-stas** [comment]
Looks good, but I'd be happy to see more context - why is relocatable different? Why `scylla_version` and `install_dir_` are mutually exclusive?

File: tests/src/integration/ccm/bridge.cpp (hunk 1 above)
Comment (fruch) [reply tests/src/integration/ccm/bridge.cpp:402]: yes we can drop it out

**Comment by fruch** [comment]
> Looks good, but I'd be happy to see more context - why is relocatable different? Why `scylla_version` and `install_dir_` are mutually exclusive?

yes the are mutually exclusive, 
or you can work from the source directory of scylla,
or you can work with relocatable package.

you can't do both at the same time.

File: tests/src/integration/ccm/bridge.cpp (hunk 1 above)
Comment (fruch) [reply tests/src/integration/ccm/bridge.cpp]: removed that part

**Review by jul-stas — APPROVED** [review]

**Pull request merged**